tests/data/profile_stability/rhel8/pci-dss.profile

```diff
@@ -1,5 +1,6 @@
 description: Ensures PCI-DSS v3.2.1 security configuration settings are applied.
 documentation_complete: true
+reference: https://www.hhs.gov/hipaa/for-professionals/index.html
```

Comment:
This reference isn't correct for pci-dss.

Reply:
You're right. It was incorrect even in the original profile. I'll fix it.